src/styles/blocks/header.scss

```diff
@@ -0,0 +1,62 @@
+.header {
+  padding: 0 $header-paddind-inline;
```

Comment:
It seems like there is a typo in the variable name $header-paddind-inline. It should be $header-padding-inline.